src/init.cpp

```diff
@@ -892,7 +895,15 @@ bool AppInit2()
     if (fServer)
         NewThread(ThreadRPCServer, NULL);
 
-    // ********************************************************* Step 12: finished
+    // ********************************************************* Step 12: start rest listenser
+
+#ifdef REST
+    uiInterface.InitMessage(_("Starting RESTful API Listener"));
+    printf("Starting RESTful API Listener\n");
+    NewThread(ThreadRESTServer, NULL);
```

Comment:
I'm unable to find the function ThreadRESTServer() anywhere. Is this still not implemented? I'm guessing so.

Reply:
It is implemented in nebliorest.cpp which is not yet included.